pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, ghc, pkgconfig, glibcLocales, coreutils, gnugrep, gnused
-, jailbreak-cabal, hscolour, cpphs, nodePackages
+, jailbreak-cabal, hscolour, cpphs, bash, nodejs
```

Comment:
Indeed, this was from a previous iteration, where I used bash in the .jsexe wrappers.